tests/TestCase/View/CellTest.php

```diff
@@ -120,10 +120,21 @@ public function testCellImplictRenderWithError(): void
      * errors into exceptions when the caller is a __toString() method.
      */
     public function testCellWithArguments(): void
+    {
+        $cell = $this->View->cell('Articles::doEcho', ['dummy', ' message']);
+        $render = "{$cell}";
+        $this->assertStringContainsString('dummy message', $render);
+    }
+
+    public function testCellWithNamedArguments(): void
     {
         $cell = $this->View->cell('Articles::doEcho', ['msg1' => 'dummy', 'msg2' => ' message']);
         $render = "{$cell}";
         $this->assertStringContainsString('dummy message', $render);
+
+        $cell = $this->View->cell('Articles::doEcho', ['msg2' => ' dummy', 'msg1' => 'message']);
+        $render = "{$cell}";
+        $this->assertStringContainsString('message dummy', $render);
```

Comment:
Should we also have a test that looks like

```php
$cell = $this->View->cell('Articles::doEcho', msg2: ' dummy', msg1: 'message');
$render = "{$cell}";
$this->assertStringContainsString('message dummy', $render');
```

Or did I misunderstand what 'named args' meant in this pull request?

Reply:
The named args usage is when invoking the cell method, not when calling `CellTrait::cell()`. Given the existing signature of `cell()` adding support for passing data using named args while keeping BC would be messy.